sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -360,9 +401,9 @@ def query_items(  # pylint:disable=docstring-missing-param
         :keyword str session_token: Token for use with Session consistency.
         :keyword Dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword Callable response_hook: A callable invoked with the response metadata.
-        :keyword int continuation_token_limit: **provisional** The size limit in kb of the
-        response continuation token in the query response. Valid values are positive integers.
-        A value of 0 is the same as not passing a value (default no limit).
+        :keyword int continuation_token_limit: The size limit in kb of the response continuation token in the query
```

Comment:
We could add a note to the change log that this feature is now considered stable.